dist/textbox/textbox.css

```diff
@@ -62,10 +62,12 @@ input.textbox__control[disabled]:-ms-input-placeholder,
 textarea.textbox__control[disabled]:-ms-input-placeholder {
   color: var(--textbox-disabled-placeholder-color, var(--color-foreground-ghost));
 }
-input.textbox__control[readonly],
-textarea.textbox__control[readonly] {
+input.textbox__control:read-only,
```

Comment:
Hmm... isn't `[readonly]` the standard html attribute? Why are we using `read-only` here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/:read-only
Its a pseudo class.

Reply:
Actually, you're right, we should be using `[readonly]` because it also affects disabled text